joint_limits:
        foot:
                has_position_limits: true
                min_position: 0.0
                max_position: 1.0
                has_velocity_limits: true
                max_velocity: 5.0
                has_acceleration_limits: true
                max_acceleration: 5.0
                has_jerk_limits: true
                max_jerk: 1.0
                has_effort_limits: false
        knee:
                has_position_limits: true
                min_position: 0.0
                max_position: 1.0
                has_velocity_limits: true
                max_velocity: 5.0
                has_acceleration_limits: true
                max_acceleration: 5.0
                has_jerk_limits: true
                max_jerk: 1.0
                has_effort_limits: false
        ass:
                has_position_limits: true
                min_position: 0.0
                max_position: 1.0
                has_velocity_limits: true
                max_velocity: 5.0
                has_acceleration_limits: true
                max_acceleration: 5.0
                has_jerk_limits: true
                max_jerk: 1.0
                has_effort_limits: false
        neck:
                has_position_limits: true
                min_position: 0.0
                max_position: 1.0
                has_velocity_limits: true
                max_velocity: 2.0
                has_acceleration_limits: true
                max_acceleration: 2.0
                has_jerk_limits: true
                max_jerk: 1.0
                has_effort_limits: false
        head:
                has_position_limits: true
                min_position: 0.0
                max_position: 1.0
                has_velocity_limits: true
                max_velocity: 1.0
                has_acceleration_limits: true
                max_acceleration: 1.0
                has_jerk_limits: true
                max_jerk: 1.0
                has_effort_limits: false

